assets/stylesheets/style.scss

```diff
@@ -30,7 +24,7 @@
 		background: $blue-wordpress;
 		border-color: $blue-dark;
 		&:hover {
-			background: $button-hover;
```

Comment:
This was the only color we used that wasn't in the Calypso's pallete. I made an executive decision.